provider/cmd/pulumi-resource-scaleway/schema.json

```diff
@@ -4,15 +4,16 @@
     "description": "A Pulumi package for creating and managing scaleway cloud resources.",
     "keywords": [
         "pulumi",
-        "scaleway"
+        "scaleway",
+        "pulumiverse"
     ],
     "homepage": "https://leebriggs.co.uk/projects#pulumi-scaleway",
     "license": "Apache-2.0",
     "attribution": "This Pulumi package is based on the [`scaleway` Terraform Provider](https://github.com/scaleway/terraform-provider-scaleway).",
-    "repository": "https://github.com/jaxxstorm/pulumi-scaleway",
+    "repository": "https://github.com/pulumiverse/pulumi-scaleway",
     "logoUrl": "https://raw.githubusercontent.com/jaxxstorm/pulumi-scaleway/master/assets/scaleway-svgrepo-com.svg",
-    "pluginDownloadURL": "https://github.com/jaxxstorm/pulumi-scaleway/releases/download/${VERSION}",
-    "publisher": "Lee Briggs",
+    "pluginDownloadURL": "https://github.com/pulumiverse/pulumi-scaleway/releases/download/${VERSION}",
```

Comment:
I know Github Releases support requires Pulumi 3.35.3 or up, however, I would like to remain consistent with the other providers in Pulumiverse. Could you update the URL to `github://api.github.com/pulumiverse`?

Reply:
Fixed